src/components/MenuBar.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Link as DefaultLink } from 'gatsby';
+import { Grid, Typography } from '@material-ui/core';
+import { styled, makeStyles } from '@material-ui/styles';
+
+import menuItems from '../services/MenuItems';
+import logo from '../assets/logo/logo.white.png';
+import { getPageUrl } from '../Routes';
+import { margins, colors, ButtonV2 as Button } from './styledComponents';
+
+const SubGrid = styled(Grid)({
+  flex: '1',
+  zIndex: '2',
+});
+
+const Link = styled(DefaultLink)({
+  color: 'inherit',
+  textDecoration: 'none',
+});
+
+const LogoContainer = styled(Grid)({ width: 'fit-content' });
+
+const Logo = styled('img')({ width: 'fit-content' });
+
+const LogoCaution = styled(Typography)({
+  fontFamily: 'Roboto',
```

Comment:
Once again, let's make these font attributes correspond to part of a theme, rather than just specifying them component-by-component.